containers/Dockerfile_prepare_db_and_params

```diff
@@ -0,0 +1,13 @@
+FROM debian:11
+LABEL authors="Athanasios Baltzis" \
```

Comment:
where are all the Docker images hosted?

Reply:
The only Docker images that we use (AF2, AF2_MSA, AF2_split, localcolabfold) are hosted in the nfcore dockerhub (https://hub.docker.com/search?q=nfcore%2Fproteinfold). I will rename the recipes filenames so that they match with the respective containers' names and I will delete the recipes that are not used anymore (replaced by the respective biocontainers).